### PR TITLE
Fixed 3 errors and caused one more.

### PR DESCRIPTION
**Pull request by Raijuu**
Happy to work with the undead.

### CODE REVIEW

File: src/Run_Circle.java [hunk 1]
```diff
@@ -22,8 +22,9 @@ public static void main(String[] args){
 		 * Observe, Constructor 2 needed data for it's double r. Here, we tell it (give r value 2.0)
 		 * r now equals 2.0, and that means that radius now equals 2.0!
 		 */
+		 String color = "Blue";
```
Comment (ElectroCamel) [inline src/Run_Circle.java]: Hm. Why not just pass "blue" directly as a parameter instead of creating a variable?

Comment (Raijuu) [inline src/Run_Circle.java]: I was getting an "unclosed literal character" error on that with the original source.

Run_Circle.java:27: error: unclosed character literal
                Circle c3 = new Circle(2.0, 'Blue');

I thought java was complaining about not using a variable.

Comment (Raijuu) [inline src/Run_Circle.java]: Oh maybe I was scrolling through the error log into old errors or tripping on something different. I just tried again and i guess double quotes does solve the problem.

Comment (HappyZombies) [inline src/Run_Circle.java]: I am using eclipse to run this. But yes, Java likes double quotes for it's String's.

Comment (ElectroCamel) [inline src/Run_Circle.java]: Yeah, for the compiler there's no difference between a string passed in quotes and via a variable.

**Comment by HappyZombies** [comment]
Is the reason that I can't merge this request is because there is a conflict ? How do I attend to that? :0

**Comment by Raijuu** [comment]
I'm guessing because I forked from this point in time:
https://github.com/HappyZombies/Circle/commit/1e1c8a80ad4f548e6f47d6536b46a257c7f51616

I'm not sure how to fix that.

**Comment by cubrr** [comment]
> Is the reason that I can't merge this request is because there is a conflict ? How do I attend to that? :0

Check this out: https://help.github.com/articles/merging-a-pull-request/#checking-out-a-pull-request-locally

**Comment by HappyZombies** [comment]
Okay, I get the idea. I'll be merging this. I'll be adding the square.java

**Pull request merged**